.github/CODEOWNERS

```diff
@@ -37,6 +37,9 @@
 
 /sdk/search/ @xirzec
 
+# Smoke Tests
+/common/smoke-test/ @ramya-rao-a @chradek @daviwil @sadasant @jeremymeng @southpolesteve
```

Comment:
Not sure about @daviwil. Is it for identity?

Reply:
Yep, I'm (unfortunately) not as active in this repo these days.

Reply:
Switched to @jonathandturner from our offline discussion